test/test_inverse.jl

```diff
@@ -20,11 +20,15 @@ InverseFunctions.inverse(f::Bar) = Bar(inv(f.A))
 
 
 @testset "inverse" begin
-    f_without_inverse(x) = x^2
+    f_without_inverse(x) = Returns(1)
```

Comment:
Is `Returns` defined in Julia 1.0? I would assume that `f_without_inverse(rand())` errors on older Julia versions which seems a bit unfortunate. Maybe use something simpler?

Reply:
Just checked:
```julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.0.5 (2019-09-09)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> f_without_inverse(x) = Returns(1) 
f_without_inverse (generic function with 1 method)

julia> f_without_inverse(rand())
ERROR: UndefVarError: Returns not defined
Stacktrace:
 [1] f_without_inverse(::Float64) at ./REPL[1]:1
 [2] top-level scope at none:0

Reply:
It's not even part of the LTS:
```julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.6.6 (2022-03-28)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> f_without_inverse(x) = Returns(1)
f_without_inverse (generic function with 1 method)

julia> f_without_inverse(rand())
ERROR: UndefVarError: Returns not defined
Stacktrace:
 [1] f_without_inverse(x::Float64)
   @ Main ./REPL[1]:1
 [2] top-level scope
   @ REPL[2]:1
```

Reply:
Oooops, yep, forgot that `Returns` is "new" ...